src/services/completions.ts

```diff
@@ -2084,6 +2084,10 @@ export function getCompletionEntriesFromSymbols(
             continue;
         }
 
+        if (!isTypeOnlyLocation && isInJSFile(sourceFile) && !symbolHasValueDeclaration(symbol)) {

```

Comment:
```suggestion
        if (!isTypeOnlyLocation && isInJSFile(sourceFile) && symbol.flags & SymbolFlags.Value) {
```

Reply:
Hmm. It looks to me like `symbol.flags` aren't being populated properly for imports from JS files. Example from ae5fc3d410:

```plaintext
  1) fourslash tests
       tests/cases/fourslash/completionsImport_require_addNew.ts
         fourslash test completionsImport_require_addNew.ts runs correctly:
     Error: At marker '': Includes: completion 'x' not found.
```

Additionally, the `jsFileImportNoTypes.ts` baseline isn't including the namespace in the imports anymore. I went back to the way it was - renaming the function to `symbolHasNoTypeOnlyDeclaration` to be a bit more accurate.

Reply:
> Hmm. It looks to me like symbol.flags aren't being populated properly for imports from JS files. 

Isn't this a problem that should be fixed? 🤔

Reply:
Looking more deeply at `tests/cases/fourslash/completionsImport_require_addNew.ts`, it's when we have a `.js` file with `module.exports`:

```js
// @allowJs: true

// @Filename: /a.js
////const x = 0;
////module.exports = { x };

// @Filename: /b.js
////x/**/
```

The flags on the symbol are:

```plaintext
__debugFlags:
'ExportDoesNotSupportDefaultModifier|Alias|AliasExcludes|ModuleMember|Classifiable|All|Assignment'
```

Note that `Value` isn't there.

Reply:
> Note that Value isn't there. Is this intentional with .js files?

No idea. Sounds off to me but perhaps there are some reasons why it does work like that 🤷